code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -158,9 +156,54 @@
 
 /obj/item/reagent_containers/glass/beaker/jar
 	name = "honey jar"
-	desc = "A jar for honey. It can hold up to 60 units of sweet delight. Unable to withstand reagents of an extreme pH."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "vapour"
+	desc = "A jar for honey. It can hold up to 50 units of sweet delight. Unable to withstand reagents of an extreme pH."
+	icon_state = "honey"
+
+/obj/item/reagent_containers/glass/beaker/glass_dish
+	name = "glass dish"
+	desc = "A tiny glass dish. It can hold up to 3 units. Unable to withstand reagents of an extreme pH."
+	materials = list(MAT_GLASS=500)
+	icon_state = "glass_disk"
+	possible_transfer_amounts = list(0.1,0.5,0.75,1,2,3)
+	volume = 3
+
+/obj/item/reagent_containers/glass/beaker/glass_dish/Initialize()
+	beaker_weakness_bitflag &= ~PH_WEAK
+	. = ..()
+
+/obj/item/reagent_containers/glass/beaker/flask_large

```

Comment:
Tell me why the `flask_large`, `flaskspouty` and `flask_small` paths shouldn't be `flask/large`, `flask/spouty` and `flask` respectively.

Reply:
Consistency

Reply:
In layman terms, common sense tells me a large flask is still a flask, so why wouldn't it be a flask subtype?
Please explain me how could ever define this little bloat of unintuitive copypasta "Consistency".

Reply:
all the different beaker types are based on the glass beaker in this file.
Even things like metamatteral being not a subtype below large.
I cant fine a single beaker thats past the subtype of the top of the file beaker

Reply:
Well past bucket is water bottle and water bottle large

Reply:
Also having them subtypes does nothing really as all the vars are different.

Reply:
Sadly I'm having hard time understanding your flow of logic here. But I'll repeat myself.
the large flask and the flask with spout should be flask subtypes for better maintainability should they share similar charateristics in the future, other than the lack of the PH_WEAK beaker bitflag.